src/miscwrappers.cpp

```diff
@@ -463,12 +467,14 @@ syscall(long sys_num, ...)
 #endif // ifdef __x86_64__
 
 # ifndef __aarch64__
+#ifndef __riscv
```

Comment:
Please add a similar FIXME comment:
`// FIXME:  For 'riscv', use `ret = pipe2(fds, 0);`

For your own information, you can google pipe2, or do 'man pipe2', for background.